modules/cvv/CMakeLists.txt

```diff
@@ -1,3 +1,6 @@
+set(the_description "Debug visualization framework")
+ocv_define_module(cvv opencv_core opencv_imgproc opencv_features2d ${CVV_LIBRARIES} WRAP python)
```

Comment:
I think this file should be reverted:

- variable `CVV_LIBRARIES` is used before being set
- other preparation steps (`add_definitions`, `include_directories`, set compiler flags)  do not affect some subtargets, probably leading to #659